Tests/TestProjectGenerator/Reqnroll.TestProjectGenerator/Data/TargetFramework.cs

```diff
@@ -20,6 +20,7 @@ public enum TargetFramework
         Net50,
         Net60,
         Net70,
-        Net80
+        Net80,
+        Net90
```

Comment:
```suggestion
        Net90,
```

This will make adding a new framework a one-line diff in the future.